src/BoundaryConditions/perturbation_advection_open_boundary_matching_scheme.jl

```diff
@@ -42,10 +54,20 @@ Adapt.adapt_structure(to, pe::PerturbationAdvection) =
                           adapt(to, pe.inflow_timescale),
                           adapt(to, pe.outflow_timescale))
 
+"""
+    PerturbationAdvectionOpenBoundaryCondition(val, FT = Float64;
+                                               backward_step = true,
+                                               outflow_timescale = Inf,
+                                               inflow_timescale = 0, kwargs...)
+
+Creates a `PerturbationAdvectionOpenBoundaryCondition` with a given `outflow_timescale` and
+`inflow_timescale`. `backward_step` determines whether we assume a backward and forward time
+discretization. For details about this method, refer to the docstring for `PerturbationAdvection`.
+"""
 function PerturbationAdvectionOpenBoundaryCondition(val, FT = Float64;
                                                     backward_step = true,
                                                     outflow_timescale = Inf,
-                                                    inflow_timescale = 300.0, kwargs...)
+                                                    inflow_timescale = 0, kwargs...)
```

Comment:
0 is not a valid choice, right? this will make the model blow up. If there's no good default, better just to make it required

Reply:
@jagoosw can correct me if I'm wrong, but `0` is a valid choice here and it means that the inflow is immediately set to the exterior condition (`val` in this case). Numerically it needs to be treated with a bit of care (hence the `ifelse` statements I included) and mathematically the limit is well-defined too.

Reply:
okay, after I wrote this I noticed that there is a check for `inflow_timescale==0` basically (although not written that way yet)